jest.config.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-undef */
+module.exports = {
+  globals: {
+    'ts-jest': {
+      tsConfig: '<rootDir/tsconfig.json>',
+    },
+  },
+  projects: [
+    {
+      displayName: 'Unit Tests',
+      testEnvironment: 'jsdom',
+      testMatch: ['**/src/tests/*.spec.ts'],
```

Comment:
Os testes estarão no próprio diretório do módulo/componente/página, não?
Se for o caso, esse caminho está incorreto.